src/parcsr_ls/par_amg.c

```diff
@@ -211,6 +214,9 @@ hypre_BoomerAMGCreate()
    ilu_max_row_nnz = 20;
    ilu_max_iter = 1;
    ilu_droptol = 0.01;
+   ilu_tri_solve = 1;
```

Comment:
@PaulMullowney  Does this mean iterative LU solve is the default?

Reply:
We can use another parameter name if you can think of something more intuitive.

Reply:
Is it direct solve? The following is from the header file. Maybe I missed something.
```
/**
 * Defines triangular solver for ILUT smoother, 0-direct, 1-iterative
 * For further explanation see description of ILU.
 **/
HYPRE_Int HYPRE_BoomerAMGSetILUTriSolve( HYPRE_Solver  solver,
										 HYPRE_Int     ilu_tri_solve);
```

Reply:
Ugh. Those comments were written a while. I'll fix. Good catch.

Reply:
I'll go through all the default values, docs and code comments and make sure tri_solve=1 means direct solve.

Reply:
@liruipeng I just pushed to my fork/branch. Can you pull that in or should I check out your branch?

Reply:
Hi @PaulMullowney Feel free to push to this branch (you should be able to, I assumed). I can pull in otherwise. Thanks!